contracts/TimeLock.sol

```diff
@@ -177,7 +183,10 @@ contract TimeLock is AragonApp, IForwarder, IForwarderFee {
             }
         }
 
-        return (lockAmount.mul(activeLocks).mul(spamPenaltyFactor).div(PCT_BASE), lockDuration.mul(activeLocks).mul(spamPenaltyFactor).div(PCT_BASE));
+        uint256 totalAmount = lockAmount.mul(activeLocks).mul(spamPenaltyFactor).div(PCT_BASE);
+        uint256 totalPenalty = lockDuration.mul(activeLocks).mul(spamPenaltyFactor).div(PCT_BASE);
```

Comment:
Maybe we could use `totalDuration` instead ?

Reply:
Yup, definitely better.